packages/bpk-component-horizontal-nav/src/BpkHorizontalNavItem.module.scss

```diff
@@ -43,6 +43,10 @@
     cursor: pointer;
     fill: currentcolor;
 
+    &:focus-visible {
+      outline-offset: -(tokens.$bpk-one-pixel-rem * 2);
```

Comment:
This is the value from the original mix-in, but negative.

Reply:
Previously, it showed just 2 lines, which is undesired. 
I agree that the colour contrast is not great, but I have just fixed the appearance so the focus indicator is around the entire button. 

We can discuss with design if we want to adjust how the focus indicator looks, but that is out of scope for this PR.

Reply:
Discussed this with the design team, and they are okay with allowing this modification on the old component.